### DISCUSSION TITLE
Seeking help  to delete my account

### DISCUSSION THREAD

**Discussion by trandanhtan1234** [community/community#133588] [Programming Help]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I wanted to delete an account mine but the delete button is disabled with a note "Please contact us if you'd like to delete your account"
So I created a ticket for that but it was like 2 months ago and still no feedback until now
What do I do now?
Please help me
Thanks
![delete_git](https://github.com/user-attachments/assets/49d23771-3590-447b-9e1e-7cec9888cdd1)

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @trandanhtan1234 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by trandanhtan1234** [reply]
Thank you kindly @queenofcorgis! So right all I have to do is wait for the devs to answer me or should I make contact with them somehow since you helped me move this Accessibility category?

Best Regards

**Comment by invictusaman** [comment]
According to [Github Documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account#deleting-your-personal-account), before you delete a personal account,

- If you're the only owner in the organization, you must transfer ownership to another person or delete your organization.
- If there are other organization owners in the organization, you must remove yourself from the organization.

Hope that helps.

**Reply by trandanhtan1234** [reply]
Thanks for your answer @invictusaman!
I read that and did as it says already and I think I did it correctly
![image](https://github.com/user-attachments/assets/a84c6272-58d4-4c8b-a9f1-331aad5d30ee)
It shows like this means I did it correctly, right?

**Reply by invictusaman** [reply]
Yes, you are not a member of any organizations. Best option might be to contact Github team again, and hope for them to reply. Sorry.

**Reply by trandanhtan1234** [reply]
I understand. Thanks for your reply by the way @invictusaman. It means a lot

**Reply by trandanhtan1234** [reply]
Just one more question. To make contact with them, all I need to do is create a ticket in this page, right?
![image](https://github.com/user-attachments/assets/8a08d5e0-e2ed-4e5c-9f56-8d799ef592fe)
Because in the other account which I want to delete I made two tickets about 2 months ago but never got a response so I want to ask Could it be different this time if I make a ticket in this account

**Reply by LiteBrite82** [reply]
Hi @trandanhtan1234 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

Thank you!

**Reply by trandanhtan1234** [reply]
Thanks for your rely @LiteBrite82 
It means a lot!!! Guess all I have to do now is wait, then

**Comment by NimsaraLiyanage** [comment]
If you're the only owner in the organization, you must transfer ownership to another person or delete your organization.
If there are other organization owners in the organization, you must remove yourself from the organization.

**Reply by DinLiren** [reply]
@NimsaraLiyanage thank you
